openapi/index_openapi.json

```diff
@@ -258,6 +258,40 @@
             "name": "include",
             "in": "query"
           },
+          {
+            "required": false,
+            "schema": {
+              "title": "First Frame",
+              "minimum": 0.0,
+              "type": "integer",
+              "default": 0
+            },
+            "name": "first_frame",
+            "in": "query"
```

Comment:
I think something has gone wrong here for this to be added to the index schema

Reply:
It automatically got added when I ran `invoke update-openapijson`
I do not really know how this schema is created. I used the same method for defining first_frame as for the other query parameters. The other query parameters also occur in `index_openapi.json` so I do not see why it is wrong for first_frame to be listed here as well.